runners/spark/2/build.gradle

```diff
@@ -22,12 +22,7 @@ project.ext {
   // Set the version of all Spark-related dependencies here.
   spark_version = '2.4.8'
   spark_scala_version = '2.11'
-
-  // Version specific code overrides.
-  main_source_overrides = ['./src/main/java']
-  test_source_overrides = ['./src/test/java']
-  main_resources_overrides = []
-  test_resources_overrides = []
+  copySourceBase = true // enabled to use Spark 3 as primary dev version
```

Comment:
Is it a typo here? Should it be `Spark2`?

Reply:
nup, Spark 3 should be the primary version for development ... so we copy sources for Spark 2

Reply:
Maybe choose another name for that then? It sounds a bit ambiguous now for me.

Reply:
👍 I've rephrased the comment